terraform/aws/modules/eks/eks-node-group.tf

```diff
@@ -0,0 +1,61 @@
+resource "aws_iam_role" "nodes_general" {
+  name = "eks-node-groupgeneral"
+  assume_role_policy = <<POLICY
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Effect": "Allow",
+      "Principal": {
+        "Service": "ec2.amazonaws.com"
+      },
+      "Action": "sts:AssumeRole"
+    }
+  ]
+}
+POLICY
+
+}
+resource "aws_iam_role_policy_attachment" "amazon_eks_worker_node_policy_general" {
+  policy_arn = "arn:aws:iam::aws:policy/AmazonEKSWorkerNodePolicy"
+  role = aws_iam_role.nodes_general.name
+}
+resource "aws_iam_role_policy_attachment" "amazon_eks_cni_policy_general" {
+  policy_arn = "arn:aws:iam::aws:policy/AmazonEKS_CNI_Policy"
+  role = aws_iam_role.nodes_general.name
+}
+resource "aws_iam_role_policy_attachment" "amazon_ec2_container_registry_read_only_policy" {
+  policy_arn = "arn:aws:iam::aws:policy/AmazonEC2ContainerRegistryReadOnly"
+  role = aws_iam_role.nodes_general.name
+}
+
+resource "aws_eks_node_group" "nodes_general" {
+  cluster_name    = aws_eks_cluster.demo.name
+  node_group_name = "nodes-general"
+  node_role_arn   = aws_iam_role.nodes_general.arn
+  subnet_ids      = [
+    aws_subnet.private_us_east_1a.id,
```

Comment:
pls pass a variable, so that this is configurable. For example tomorrow if I want to create it in 4 subnets instead of 2, I should only need to change my variable.